### PR TITLE
FAQ: explain that we can't delete accounts

### LINKED ISSUE #555: Account deletion link
should link to FAQ entry explaining that we have no accounts

maybe this could help some people. what do you think?

### PR DESCRIPTION
**Pull request by missytake**
fix #555

(this is also meant as a test for #679 - please only merge after we confirmed opening this PR didn't delete `/var/www/html/staging`. After merging we can check whether merging it causes #679) 

CC @link2xt @brabo

### CODE REVIEW

File: en/help.md [hunk 1]
```diff
@@ -182,6 +182,18 @@ to mute a chat, use the chat's menu (Android/Desktop) or the chat's profile (iOS
   message them or write to a group they're in as well.
 
 
+### How can I delete my account?
+
+- As you use an e-mail account for Delta Chat,
```
Comment (link2xt) [inline en/help.md]: I don't think it should be a list, these items are just sentences/paragraphs.

File: en/help.md [hunk 2]
```diff
@@ -182,6 +182,18 @@ to mute a chat, use the chat's menu (Android/Desktop) or the chat's profile (iOS
...
+  it depends on your e-mail provider.
+- We don't have any control over your e-mail account,
+  so unfortunately we can't help you with that :/
+- If you want to keep the account,
+  but uninstall Delta Chat,
+  it is recommended to leave any group chat before uninstalling Delta Chat.
+  Otherwise you might receive a lot of encrypted, unreadable mail in the future.
```
Comment (link2xt) [inline en/help.md]: You can also export the key, the messages don't have to be unreadable.

File: en/help.md [hunk 3]
```diff
@@ -182,6 +182,18 @@ to mute a chat, use the chat's menu (Android/Desktop) or the chat's profile (iOS
   message them or write to a group they're in as well.
 
 
+### How can I delete my account?
+
+- As you use an e-mail account for Delta Chat,
+  it depends on your e-mail provider.
```
Comment (link2xt) [inline en/help.md]: What is "it"?

**Comment by missytake** [comment]
The first commit didn't delete the directory, no it created a `680` subdirectory, as expected.

**Review by link2xt — APPROVED** [review]

**Comment by missytake** [comment]
The 2nd and 3rd commits also didn't cause problems.

**Comment by missytake** [comment]
I think I found the reason for #679:

now after merging, there is the file `/var/www/html/staging/index.html`, which contains `This preview is outdated and has been removed.`

This should be in `/var/www/html/staging/680/index.html` instead.

**Pull request merged**